src/engine/controls/bpmcontrol.cpp

```diff
@@ -184,26 +191,16 @@ void BpmControl::slotAdjustBeatsSlower(double v) {
 }
 
 void BpmControl::slotTranslateBeatsEarlier(double v) {
-    if (v <= 0) {
-        return;
-    }
-    const TrackPointer pTrack = getEngineBuffer()->getLoadedTrack();
-    if (!pTrack) {
-        return;
-    }
-    const mixxx::BeatsPointer pBeats = pTrack->getBeats();
-    if (pBeats) {
-        const double sampleOffset = frameInfo().sampleRate * -0.01;
-        const mixxx::audio::FrameDiff_t frameOffset = sampleOffset / mixxx::kEngineChannelCount;
-        const auto translatedBeats = pBeats->tryTranslate(frameOffset);
-        if (translatedBeats) {
-            pTrack->trySetBeats(*translatedBeats);
-        }
-    }
+    slotTranslateBeatsMove(v);
```

Comment:
```suggestion
    if (v <= 0) {
        return;
    }
    slotTranslateBeatsMove(-1.0);
```
Otherwise the code is identical to `slotTranslateBeatsLater` and the behavior of the function would be different, than before.

Reply:
:facepalm: 
Thank you. I pushed the fix:
later: +1
earlier: -1